devolutions-session/src/main.rs

```diff
@@ -19,15 +19,17 @@ fn main() -> anyhow::Result<()> {
 
     std::fs::create_dir_all(get_data_dir()).context("Failed to create data directory")?;
 
-    let config = ConfHandle::init().context("Failed to initialize configuration")?;
+    let conf = ConfHandle::init()
+        .context("Failed to initialize configuration")?
```

Comment:
nitpick/style: You’re not behind this line of code, but I would like to mention that this should start by a lower case. It’s a nitpick as it’s not really causing much harm outside of error messages a bit less nice-looking.

Recommended read: https://github.com/Devolutions/IronRDP/blob/f5dd282271df0084f589f4e163459b41cc2f2097/STYLE.md#formatting-of-error-messages

Reply:
Addressed in [b769880](https://github.com/Devolutions/devolutions-gateway/pull/1068/commits/b769880362f9c79c02bcc8c139b318fc1429b1d6).